R/UnsupervisedTask.R

```diff
@@ -1,6 +1,6 @@
 makeUnsupervisedTask = function(type, data, weights = NULL, blocking = NULL, fixup.data = "warn", check.data = TRUE) {
   task = makeTask(type, data, weights, blocking, fixup.data = fixup.data, check.data = check.data)
-  checkTaskData(getTaskData(data))
+  checkTaskData(task$env$data)
```

Comment:
please really read the threads, issue + PR for stuff like this, to we dont waste time.

but more to the point: the object is not completely constructed at this point in time i guess. so the getter does not work. and here, as an exception, it should then be OK to directly access the slot.

errrrr: the previous line getTaskData(data) seeems to be complete bullshit? "data" is not a task here?